app/assets/javascripts/google-analytics.js

```diff
@@ -0,0 +1,47 @@
+// Tracks special user actions with Google Analytics
```

Comment:
How are we tracking...regular use actions? (I'm thinking about the filenaming here - are we importing the Google Analytics JS using another gem, etc.?)